rllib/core/learner/learner.py

```diff
@@ -776,40 +810,30 @@ def additional_update_per_module(self, module_id: ModuleID, tau: float):
 
     @OverrideToImplementCustomLogic_CallToSuperRecommended
     def additional_update_per_module(
-        self, module_id: ModuleID, **kwargs
+        self, module_id: ModuleID, timestep: int, **kwargs
     ) -> Dict[str, Any]:
         """Apply additional non-gradient based updates for a single module.
 
         See `additional_update` for more details.
 
         Args:
             module_id: The id of the module to update.
+            timestep: The current global timestep (to be used with schedulers).
             **kwargs: Keyword arguments to use for the additional update.
 
         Returns:
             A dictionary of results from the update
         """
-        return {}
-
-    @OverrideToImplementCustomLogic
-    def postprocess_gradients(
-        self,
-        gradients_dict: Mapping[str, Any],
-    ) -> Mapping[str, Any]:
-        """Applies potential postprocessing operations on the gradients.
-
-        This method is called after gradients have been computed, and modifies them
-        before they are applied to the respective module(s).
-        This includes grad clipping by value, norm, or global-norm, or other
-        algorithm specific gradient postprocessing steps.
-
-        Args:
-            gradients_dict: A dictionary of gradients.
+        results = {}
+        # Handle lr scheduling updates and apply new learning rates to the optimizers.
```

Comment:
Moved here from the Torch|TfLearner implementations (no longer necessary to keep this logic in framework-specific subclasses). We use the new `self._set_optimizer_lr()` API for this.